tfs/pom.xml

```diff
@@ -206,6 +206,18 @@
       <version>1.25</version>
     </dependency>
     <dependency>
+      <!-- Used for configuring collections globally -->
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-job</artifactId>
```

Comment:
do you really need hard dependency on concrete job type? Could it be optional dep?